tests/mysql/query.rs

```diff
@@ -540,27 +540,30 @@ fn select_36() {
 
 #[test]
 fn select_37() {
-    let (statement, values) = sea_query::Query::select()
+    let (statement, values) = Query::select()
         .column(Glyph::Id)
         .from(Glyph::Table)
         .cond_where(Cond::any().add(Cond::all()).add(Cond::any()))
-        .build(sea_query::MysqlQueryBuilder);
+        .build(MysqlQueryBuilder);
 
-    assert_eq!(statement, r#"SELECT `id` FROM `glyph`"#);
+    assert_eq!(
+        statement,
+        r#"SELECT `id` FROM `glyph` WHERE (TRUE) OR (FALSE)"#
+    );
```

Comment:
Can we add a negation test as well?
`Cond::all().add(Cond::all().not()).add(Cond::any().not()).not()`

Should turn into `NOT (NOT TRUE OR NOT FALSE)`

Reply:
@nitnelave done!